File: WebApp/src/app/identity/security-sessions/sessions-management/contracts/session-summary.dto.ts
export interface SessionSummaryDto {
    userId: string;
    id: string;
    deviceName: string;
    loginAt: Date;
    lastSeenAt: Date;
    logoutAt: Date | null;
    counrty: string;
    city: string;
    durationMinutes: number;
    isRevoked: boolean;
    revokedReason: string | null;
}

File: backend/SNS.Application/Identity/SecuritySessions/SessionsManagement/Queries/GetUserSessions/SessionSummaryDto.cs
namespace SNS.Application.Identity.SecuritySessions.SessionsManagement.Queries.GetUserSessions;

public sealed record SessionSummaryDto(
    Guid UserId,
    Guid Id,
    string DeviceName,
    DateTime LoginAt,
    DateTime LastSeenAt,
    DateTime? LogoutAt,
    string Counrty,
    string City,
    int DurationMinutes,
    bool IsRevoked,
    string? RevokedReason);
